src/client/clientauth.h

```diff
@@ -0,0 +1,52 @@
+/*
+Minetest
+Copyright (C) 2024 SFENCE, <sfence.software@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#pragma once
+
+#include <string>
+#include "network/networkprotocol.h"
+
+class ClientAuth
+{
+public:
+	ClientAuth();
+	ClientAuth(const std::string &player_name, const std::string &password);
+	~ClientAuth();
```

Comment:
As far as I know, according to the [rule of three/five/zero](https://en.cppreference.com/w/cpp/language/rule_of_three), the destructor shouldn't be alone.

Reply:
Fixed.